starfish/core/imagestack/test/imagestack_test_utils.py

```diff
@@ -41,6 +40,8 @@ def verify_physical_coordinates(stack: ImageStack,
     # If zplane provided, test expected_z_coordinates on specific plane.
     # Else just test expected_z_coordinates on entire array
     if zplane is not None:
-        assert np.isclose(stack.xarray[Coordinates.Z.value][zplane], expected_z_coordinates)
+        assert np.isclose(
+            stack.xarray.sel({Axes.ZPLANE.value: zplane})[Coordinates.Z.value],
```

Comment:
I don't think you need to do the sel, stack.xarray[Coordinates.Z.value].values should give you the array